src/style.css

```diff
@@ -0,0 +1,77 @@
+@import "./reset.css";
+@import "./variables.css";
+
+@import "./styles/product-card.css";
+@import "./styles/stars.css";
+
+.body {
+  margin: 0;
+}
+
+.image {
+  width: 160px;
+  height: 134px;
+}
+
+.title {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 150%;
+  color: var(--color-dark);
+  display: flex;
+}
+
+.product-code {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 140%;
+  color: var(--color-gray);
+}
+
+.price {
+  display: flex;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 112%;
+  color: var(--color-dark);
+  text-align: right;
+  justify-content: space-between;
+}
+
+.price::before {
+  content: "Price:";
```

Comment:
it should be `html` content

Reply:
i dong agree with u, because its one element "price" and this content won't change because on cards we will always have price and label price before. And we can easily take element price and css rule price::before to add this element to another card